src/sidebar/reducers/selection.js

```diff
@@ -132,6 +133,17 @@ var update = {
     };
   },
 
+  ADD_ANNOTATIONS(state, action) {
+    var counts = tabs.counts(action.annotations);
```

Comment:
`tabs.counts` works but it also computes counts that we don't actually use - such as orphans and annotations. It would be slightly more efficient to just use the same code that `tabs.js` uses to compute the `notes` property: `countIf(annotations, metadata.isPageNote)`.